<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModelBuilder</class>
 <widget class="QDialog" name="ModelBuilder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>888</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ModelBuilder</string>
  </property>
  <widget class="QPushButton" name="pushButton_Execute">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>470</y>
     <width>151</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Execute</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_path">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>621</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Data source...</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>701</width>
     <height>451</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_target">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>420</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>LogEC</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="placeholderText">
    <string>&lt;value&gt;, &lt;value&gt;, &lt;value&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>400</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Target:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_pickle">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>420</y>
     <width>71</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Pickle</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_find">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>470</y>
     <width>71</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>40</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Validation</string>
   </property>
   <widget class="QRadioButton" name="radioButton_validation_CV">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Random folds:</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_validation_year">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Year:</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_year">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>2010</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>year...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_CV_folds">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>year...</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>290</y>
     <width>161</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Learning Method</string>
   </property>
   <widget class="QRadioButton" name="radioButton_PLS">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>PLS regression</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Boost">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Trees w/ boosting</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Logistic">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Logistic regression</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>200</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Threshold, balance method:</string>
   </property>
   <widget class="QCheckBox" name="checkBox_Counts">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Counts</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_Proportions">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Proportions</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>130</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Specificities</string>
   </property>
   <widget class="QLineEdit" name="lineEdit_specificities">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>0.89, 0.90, 0.91</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>&lt;value&gt;, &lt;value&gt;, &lt;value&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
